ui/app/AppLayouts/Wallet/popups/AddAccountModal.qml

```diff
@@ -150,6 +150,12 @@ StatusModal {
                         popup.passwordValidationError = ""
                         _internal.waitTimer.restart()
                     }
+                    onKeyPressed: {
```

Comment:
Would be much easier with something like: `KeyNavigation.tab: accountNameInput.input.edit`

Aka, too much imperative JS code :)

Reply:
But not sure if the Input field wouldn't already eat the TAB key event

Reply:
It sadly does eat up! keys.onPressed is propagated by a signal.